tools/set_project_event_subs.php

```diff
@@ -7,12 +7,19 @@
 require_login();
 
 $projectid = validate_projectID('projectid', @$_POST['projectid']);
+$project = new Project($projectid);
 $return_uri = urldecode($_POST['return_uri']);
 
 $subs = array();
 $unsubs = array();
 foreach ( $subscribable_project_events as $event => $label )
 {
+    // Disallow subscribing to SR report posted event for anyone who is not a PP
+    if ($event == 'sr_reported' && !$project->PPer_is_user($pguser))
```

Comment:
It's worth noting that this is duplicated code, the same code is used in `project.php` ([here](https://github.com/DistributedProofreaders/dproofreaders/blob/master/project.php#L1142-L1145)), so it might be worthwhile to turn that condition into a function `can_user_subscribe_to_event($username, $projectid, $event)` to future proof it.

Reply:
For the comment, it's more correct to say "for anyone who is not the PP for the project"

Reply:
@mlazaric True, although I'm not going to block this merging because of that. I will merge this unless you want to do another commit to abstract that logic out into a function.